tests/outputs/amsr2_ocean.tc.windspeed.imagery_clean/20200518_073601_IO012020_amsr2_gcom-w1_windspeed_140kts_85p45_1p0-clean.png.yaml

```diff
@@ -8,8 +8,8 @@ bounding_box:
   minlon: 79.90497403967373
   pixel_height_m: 1000.0
   pixel_width_m: 1000.0
-  proj4_string: +proj=eqc +lat_ts=0 +lat_0=13.4 +lon_0=86.2 +x_0=0 +y_0=0 +R=6371228
-    +units=m +no_defs +type=crs
+  proj4_string: +R=6371228 +lat_0=13.4 +lat_ts=0 +lon_0=86.2 +no_defs +proj=eqc +type=crs
```

Comment:
Was this a pyres Maple version update?  Do we know they are not going to just switch right back with someone else's install ?